rulette-mysql-provider/src/main/java/com/github/kislayverma/rulette/mysql/MysqlDataProvider.java

```diff
@@ -241,11 +273,21 @@ public RuleSystemMetaData loadRuleSystemMetaData(String ruleSystemName) throws E
                     ? RuleInputType.VALUE : RuleInputType.RANGE;
             String dataType = resultSet.getString("data_type").toUpperCase();
 
-            inputs.add(new RuleInputMetaData(resultSet.getInt("id"),
-                    resultSet.getString("name"),
-                    resultSet.getInt("priority"),
-                    ruleType,
-                    dataType));
+            if(ruleType != RuleInputType.RANGE) {
```

Comment:
We should always have upper and lower bound columns in the schema. No reason to do this.